app/app.go

```diff
@@ -586,6 +595,15 @@ func (app *App) ModuleAccountAddrs() map[string]bool {
 	return modAccAddrs
 }
 
+// BlockedModuleAccountAddrs returns all the app's blocked module account
+// addresses.
+func (app *App) BlockedModuleAccountAddrs() map[string]bool {
+	modAccAddrs := app.ModuleAccountAddrs()
+	delete(modAccAddrs, authtypes.NewModuleAddress(govtypes.ModuleName).String())
+
+	return modAccAddrs
+}
+
```

Comment:
Where did you get this `BlockedModuleAccountAddrs` function from?

Reply:
This function is used in the call of `bankkeeper.NewBaseKeeper()` in `app.go` which takes a list of blocked accounts.  This function just removes the `x/gov` module from the list of blocked accounts, so it can potentially receive funds for proposals etc.  This change only affects the `app` used in things like testing and simulation for this repository.  No actual impact on the `x/fundraising` module.

Reply:
The reason why I asked is because Cosmos SDK v0.46.0 `simapp` as well as `spn` don't seem to use `BlockedModuleAccountAddrs()`, they both use `ModuleAccountAddrs()`. 
- https://github.com/cosmos/cosmos-sdk/blob/v0.46.0/simapp/app.go#L519-L527
- https://github.com/tendermint/spn/blob/develop/app/app.go#L393

Reply:
Ah, I see.  I took it from [here](https://github.com/cosmos/gaia/blob/58b92fb5cea102a6576fa9aa63f181422a559133/app/app.go#L333).  Would you like to revert this?

Reply:
Just to follow the pattern of the Cosmos SDK's `simapp`, I'll revert it and merge it.